nservicebus/handlers/handler-ordering.md

```diff
@@ -21,10 +21,12 @@ The inference here is that the remaining handlers (not specified in the order) a
 
 <!-- import HandlerOrderingWithCode -->
 
-### Specifying First with ISpecifyMessageHandlerOrdering
+If you have been using `ISpecifyMessageHandlerOrdering` in a separate assembly scanned by NServiceBus we suggest to use `INeedInitialization`
```

Comment:
would prefer not to document this here. this is documentation for INeedInitialization not handler ordering. theoretically this statement could apply to any of the configuration API and it is just noise to add it to every page